CMakeLists.txt

```diff
@@ -6,7 +6,7 @@ set(CMAKE_CXX_STANDARD_REQUIRED ON)
 set(CMAKE_CXX_EXTENSIONS OFF)
 
 option(USE_GDBSTUB "Build Vita3K with GDB Debugger." OFF)
-option(USE_DISCORD_RICH_PRESENCE "Build Vita3K with Discord Rich Presence" ON)
+option(USE_DISCORD_RICH_PRESENCE "Build Vita3K with Discord Rich Presence" OFF)
```

Comment:
I'm not sure that you should default this option to OFF, as most people use windows where this is functional.

Reply:
For building the first time, I think it is important that people should able to build without trouble. Failing to build without knowing why is quite frustrating.

Reply:
I agree that this should be off. It was off initially too.